File: aroundHubSpringBoot/src/main/java/com/example/aroungHubJavaSpringBoot/aroundHubSpringBoot/controller/DeleteController.java
package com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.controller;

import org.springframework.web.bind.annotation.DeleteMapping;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/v1/get-api")
public class DeleteController {

    // http://localhost:8080/api/v1/get-api/delete/{String 값}
    @DeleteMapping(value = "/delete/{variable}")
    public String DeleteVariable(@PathVariable String variable){
        return variable;
    }
}

File: aroundHubKotlinSpringBoot/src/main/kotlin/com/example/aroungHubKotlinSpringBoot/aroundHubKotlinSpringBoot/controller/DeleteController.kt
package com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.controller

import org.springframework.web.bind.annotation.DeleteMapping
import org.springframework.web.bind.annotation.PathVariable
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/v1/get-api")
class DeleteController {

    // http://localhost:8080/api/v1/get-api/delete/{string 값}
    @DeleteMapping("/delete/{variable}")
    fun DeleteVariable(@PathVariable variable: String) : String
    {
        return variable
    }
}